pages/agent/v3/windows.md

```diff
@@ -89,6 +89,15 @@ nssm start buildkite-agent
 nssm status buildkite-agent
 ```
 
+If you'd like to change the user the buildkite-agent service runs as, you can use the same third-party tool [nssm](https://nssm.cc/) using the command-line:
+
+```
+# Change ComputerName to the name of the System under "Settings"
+# Change AccountName to the name of the Account you'd like to use
+# Change password to the password of the account you provided in the above comment
```

Comment:
"The above comment" is a bit ambiguous. Perhaps:

```suggestion
# Change password to the password of the account you'd like to use
```

What do you think?

Reply:
100%, great change

Reply:
But, keep in mind the password has to be the password already determined for the account the user is wanting to use. So if the Admin account is PC\Administrator with a password of "notarealpassword", the password field in the CLI command has to be "notarealpassword" or some kind of variable/env var to match the account password. It's not meant to be something to set the password itself. So we might wanna reconsider the wording here again.

Reply:
I suspect the formatting is working against us here. I'm going to suggest a different style.